emu/src/render/mod.rs

```diff
@@ -1,2 +1,27 @@
 /// This module contains all the data structures used to render the GBA display.
-pub mod palette_color;
+pub mod color;
+pub mod lcd;
+pub mod ppu;
+
+/// GBA display width
+pub const DISPLAY_WIDTH: usize = 240;
+
+/// GBA display height
+pub const DISPLAY_HEIGHT: usize = 160;
```

Comment:
This depends on the BG mode right? For example BG Mode 5 is `160x128` pixels

Reply:
Yes, but I want to keep it as the real size of LCD and to add for BG_MODE_5 other values referring to GBC display.